src/scss/components/_form-field.scss

```diff
@@ -0,0 +1,20 @@
+@use 'microscope-sass/lib/bem';
+
+@import '@utrecht/components/dist/form-field/css/index.css';
+
+// Overrides of the utrecht form field styles for our own theme
+.openforms-theme {
+  .utrecht-form-field {
+    @include bem.modifier('openforms') {
+      background-color: var(--of-utrecht-form-field-background-color, unset);
```

Comment:
Why have both "of" and "utrecht" in the token?

Reply:
`of` for our own namespace, as it's a custom design token that doesn't exist for the utrecht component, and the `utrecht` to relate it back to the `utrecht-form-field` component